lib/magma/commands.rb

```diff
@@ -127,4 +127,26 @@ def setup(config)
       Magma.instance.load_models
     end
   end
+
+  class Unload < Etna::Command
+    usage 'Run Unloader to dump the dataset of a model into a gzipped tsv.
+      *args - arg[0]: project_name, arg[1]: model_name, args[2]: gzip?, arg[3]: file'
```

Comment:
The usage i've been favoring is like:
`<project_name> <model_name> <gzip?> <file> # Dump the dataset of the model into a gzipped tsv`